app/helpers/elasticsearch_helper.rb

```diff
@@ -0,0 +1,35 @@
+module ElasticsearchHelper
+  def prefix_match(model, field, prefix, condition)
+    search_params = { query: { query_string: { query: "#{prefix}*", fields: [field] } } }
+    results = if Rails.env == "test"
+                # Return all records. Tests can't use elasticsearch.
```

Comment:
Why cant test use?

Reply:
We need to find a way (can be done later) to get rid of all the 'if test' conditions.

Reply:
ok let's do it later. We only really use tests for access control

Reply:
Agree, this is not great for code organization. Right now, I struggle to think of the best way to handle the issue without too much overhead. Will make a task to reevaluate this